script/commands/migrate.ts

```diff
@@ -0,0 +1,149 @@
+import { Command } from "commander";
+import { CodeDetails } from "@cosmjs/cosmwasm-stargate";
+
+import { CONTAINER, Dependencies } from "../shared/ioc";
+import { ContractNames } from "../shared/contract";
+import { ContextHook, ContextIsm } from "../shared/context";
+import { askQuestion, waitTx } from "../shared/utils";
+
+export const migrateCmd = new Command("migrate")
+  .description("Migrate contracts")
+  .configureHelp({ showGlobalOptions: true })
+  .option("-c --contracts <contracts...>", "specify contracts to migrate")
```

Comment:
How do you pass multiple? comma separated?

Reply:
Be clear that you are looking for types here

Reply:
https://github.com/tj/commander.js?tab=readme-ov-file#variadic-option like this